build/squire-raw.js

```diff
@@ -3598,11 +3598,19 @@ function Squire ( root, config ) {
     // } else {
       this.addEventListener( 'keyup', this._updatePathOnEvent );
       this.addEventListener( 'mouseup', this._updatePathOnEvent );
+      this._selectionClick = false
+      this.addEventListener('mousedown', function (e) {
+        var range = this.getSelection()
```

Comment:
I modified this to use the internal getSelection available to squire rather than resorting to _doc.  Was there a reason you did so?